code/datums/progressbar.dm

```diff
@@ -2,39 +2,44 @@
 #define PROGRESSBAR_ANIMATION_TIME 5
 
 /datum/progressbar
-	///The progress bar visual element.
+	/// The progress bar visual element.
 	var/image/bar
-	///The target where this progress bar is applied and where it is shown.
+	/// The target where this progress bar is applied and where it is shown.
 	var/atom/bar_loc
-	///The mob whose client sees the progress bar.
+	/// The mob whose client sees the progress bar.
 	var/mob/user
-	///The client seeing the progress bar.
+	/// The client seeing the progress bar.
 	var/client/user_client
-	///Effectively the number of steps the progress bar will need to do before reaching completion.
+	/// Effectively the number of steps the progress bar will need to do before reaching completion.
 	var/goal = 1
-	///Control check to see if the progress was interrupted before reaching its goal.
+	/// Control check to see if the progress was interrupted before reaching its goal.
 	var/last_progress = 0
-	///Variable to ensure smooth visual stacking on multiple progress bars.
+	/// Variable to ensure smooth visual stacking on multiple progress bars.
 	var/listindex = 0
-	///The type of our last value for bar_loc, for debugging
+	/// The type of our last value for bar_loc, for debugging
 	var/location_type
-	///Where to draw the progress bar above the icon
+	/// Where to draw the progress bar above the icon
 	var/offset_y
+	/// The progress visible to other players
+	var/obj/effect/overlay/vis/cog
+	/// The blank image to hide the cog
+	var/image/blank
```

Comment:
dumb idea just dropped, couldn't we use the bar image to override the cog? or am I bein dumb

Reply:
correct! this is what i was thinking earlier. we would need to invert it, sort of